controllers/front/default.php

```diff
@@ -129,7 +129,7 @@ protected function ajaxProcessAddComment()
 			else
 			{
 				$result = false;
-				$errors[] = $module_instance->l('Please wait before posting another comment').' '.Configuration::get('PRODUCT_COMMENTS_MINIMAL_TIME').' '.$module_instance->l('seconds before posting a new comment');
+				$errors[] = $module_instance->l('Please wait before posting another comment', 'default').' '.Configuration::get('PRODUCT_COMMENTS_MINIMAL_TIME').' '.$module_instance->l('seconds before posting a new comment', 'default');
```

Comment:
Hi,

You can't access the translation by "installed module translations" ?

Best regards.